missing package - 4.14 - don't know how to add it to docker instance...

Getting this error when trying to update to 4.14. Elwood replies to me thus: 

everyone using alpine linux is getting this error, You need to install the package linux-headers. I'm amazed this is not included by default.

Is there a way I can rebuild the docker image to include this package, or is it something you have to do? 


![Image](https://github.com/user-attachments/assets/51775cdf-bce0-4b71-8298-c26d894620b4)

Build log error reference:
```
#24 56.44 g++ -IArduinoLib -IwsServer/include -Izlib-hc -I. -g -O2 -Wall -pthread -std=c++17 -D_WEB_ONLY -D_CLOCK_1600x960   -c -o ESP8266WiFi.o ESP8266WiFi.cpp
#24 56.78 ESP8266WiFi.cpp:23:10: fatal error: linux/wireless.h: No such file or directory
#24 56.78    23 | #include <linux/wireless.h>
#24 56.78       |          ^~~~~~~~~~~~~~~~~~
#24 56.78 compilation terminated.
#24 56.79 make[1]: *** [<builtin>: ESP8266WiFi.o] Error 1
#24 56.79 make[1]: *** Waiting for unfinished jobs....
#24 57.62 make[1]: Leaving directory '/hamclock/ESPHamClock/ArduinoLib'
#24 57.62 make: *** [Makefile:148: hclibs] Error 2
#24 ERROR: process "/bin/sh -c make -j 4 hamclock-web-${HAMCLOCK_RESOLUTION}" did not complete successfully: exit code: 2

#35 [linux/arm/v7 11/13] RUN make -j 4 hamclock-web-1600x960
#35 34.81 g++ -IArduinoLib -IwsServer/include -Izlib-hc -I. -g -O2 -Wall -pthread -std=c++17 -D_WEB_ONLY -D_CLOCK_1600x960   -c -o OTAupdate.o OTAupdate.cpp
#35 CANCELED
```

Via: https://github.com/ChrisRomp/hamclock-docker/actions/runs/13480582420/job/37665242228

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,7 +10,7 @@ ARG HAMCLOCK_RESOLUTION=1600x960
 
 # Install updates and required packages
 RUN apk update && apk upgrade
-RUN apk add curl make g++ libx11-dev perl
+RUN apk add curl make g++ libx11-dev perl linux-headers
 
 RUN mkdir /hamclock
 WORKDIR /hamclock

